### ISSUE TITLE
Edit comment box not resizing 

### ISSUE THREAD

**Issue by dylanking42** [thepracticaldev/dev.to#1554]
Browser extensions disabled.

**Describe the bug**
I believe that the edit comment functionality available at the URL:
https://dev.to/**username**/comment/**commentID**/edit 
is not letting me resize the edit comment box to display multi line comments. 

**To Reproduce**

1. Write a multi line comment and post it.
2. Click on 'edit comment'
3. the page does not look like what I believe it is intended to look like.

**Expected behavior**
I would expect the edit comment functionality to allow me to resize the window, or at least expand the edit comment window text area box to fill the page. 
Behavior similar to this was mentioned in #47 which makes me believe this is an error.

**Screenshots**
Notice the large gap in between the footer and the text comment edit box.

![editing comment example](https://user-images.githubusercontent.com/15676450/51194412-01386400-18b9-11e9-8324-62d2442b3748.PNG)

![edit comment exmaple 2](https://user-images.githubusercontent.com/15676450/51194755-b3702b80-18b9-11e9-98c5-6e449ec6aa02.PNG)

**Desktop (please complete the following information):**

- OS: [e.g. iOS] Windows Version	10.0.16299 Build 16299
- Browser [e.g. chrome, safari]  FF 64.0 32 bit // IE 11.0.101

**Smartphone (please complete the following information):**
n/a

**Additional context**

This was tested on both vertical and horizontal monitor orientations. The text edit window is still rather small by default.

**Comment by jessleenyc** [comment]
Thanks for the issue! We kind of put the comment permalink page up hastily and haven't given it much love since. Currently, this is expected behavior but this is issue is approved if anyone wants to make an enhancement. Some thoughts: 
- Have the comment box auto expand as more text is included -- the same way we currently handle this on the article page w/ comments. 
- OR, since this is the comment show page, we could have a bigger comment box instead.

**Comment by willamesoares** [comment]
Hey @jessleenyc, is this being worked on by someone? I can take a look at it this week. Let me know.

**Comment by jessleenyc** [comment]
Nope, go for it! I think the second solution -- have it be a bigger comment box probably makes the most sense here.

**Comment by willamesoares** [comment]
@jessleenyc I have a question: for the text area element, why are users supposed to not be allowed to resize it? Was that on purpose?

I checked both options and from what I see it seems that the easiest way to handle edge cases and avoid future complaints about this is to allow users to resize the text area only *vertically* as much as they want.

I added a GIF to the PR [#1866](https://github.com/thepracticaldev/dev.to/pull/1866)
Let me know what you think.
